File: weisi/src/main/java/com/cjq/lib/weisi/protocol/GuomiEsbAnalyzer.java
package com.cjq.lib.weisi.protocol;

/**
 * Created by CJQ on 2018/3/22.
 */

public class GuomiEsbAnalyzer extends EsbAnalyzer {

    @Override
    public float analyzeBatteryVoltage(byte voltage, int sensorAddress) {
        return voltage * BATTERY_VOLTAGE_COEFFICIENT;
    }
}

File: wsb/src/main/java/com/wsn/lib/wsb/protocol/GuomiEsbAnalyzer.kt
package com.wsn.lib.wsb.protocol

/**
 * Created by CJQ on 2018/3/22.
 */

class GuomiEsbAnalyzer : EsbAnalyzer() {

    override fun analyzeBatteryVoltage(voltage: Byte, sensorAddress: Int): Float {
        return voltage * Analyzable.BATTERY_VOLTAGE_COEFFICIENT
    }
}
